app/controllers/Authentications.php

```diff
@@ -21,7 +21,8 @@ public function login() {
             if($_SERVER['REQUEST_METHOD'] == 'POST') {
                 // sanitize post data
                 // filter_input_array() returns false if POST var is set to scalar value
-                $_POST = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+                // $_POST = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+                $_POST = filter_input_array(INPUT_POST);
```

Comment:
Can you explain the reason for removing `FILTER_SANITIZE_STRING` in `$_POST = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);`